src/widgets/buttons.ts

```diff
@@ -723,6 +726,67 @@ export function renderAsRow (dom: HTMLDocument, pred: NamedNode, obj: NamedNode,
   ;(tr as any).subject = obj
   return tr
 }
+/* A helper function for renderAsDiv
+*  creates the NameDiv for the person
+*  Note: could not move it to the helper file because they call exported functions
+*  from buttons
+*  @internal exporting this only for unit tests
+*/
+export function createNameDiv (dom: HTMLDocument, div: HTMLDivElement, title: string | undefined, obj: NamedNode) {
+  const nameDiv = div.appendChild(dom.createElement('div'))
+  if (title) {
+    nameDiv.textContent = title
+  } else {
+    setName(nameDiv, obj) // This is async
+  }
+}
+/* A helper function for renderAsDiv
+*  creates the linkDiv for the person
+*  Note: could not move it to the helper file because they call exported functions
+*  from buttons
+* @internal exporting this only for unit tests
+*/
+export function createLinkDiv (dom: HTMLDocument, div: HTMLDivElement, obj: NamedNode, options: RenderAsDivOptions) {
+  const linkDiv = div.appendChild(dom.createElement('div'))
+  linkDiv.setAttribute('style', linkDivStyle)
+
+  if (options.deleteFunction) {
+    deleteButtonWithCheck(dom, linkDiv, options.noun || 'one', options.deleteFunction)
+  }
+
+  if (obj.uri) {
+    // blank nodes need not apply
+    if (options.link !== false) {
+      createLinkForURI(dom, linkDiv, obj)
+    }
+    dragAndDrop.makeDraggable(div, obj)
+  }
+}
+/**
+ * A Div to represent a draggable person, etc in a list
+ * configurable to add an onClick listener
+ */
+export function renderAsDiv (dom: HTMLDocument, obj: NamedNode, options: RenderAsDivOptions): HTMLElement {
```

Comment:
**suggestion** let's also add a story to storybook for this new component

Reply:
I'll need your help with this i think or someone who knows storybook.

Reply:
We could have a call this weekend. Does that fit for you?

Reply:
If you have time that would be great! I know you're busy though, if you don't end up with time maybe you can point me to one of your PRs where you did one and I can have a look?